docs/reference/search/search.asciidoc

```diff
@@ -214,35 +218,184 @@ include::{docdir}/rest-api/common-parms.asciidoc[tag=timeout]
   <<query-dsl,Query DSL>>.
 
 
+[[search-api-response-body]]
+==== {api-response-body-title}
+
+`took`::
++
+--
+(Integer)
+Milliseconds it took {es} to execute the request.
+
+This value is calculated by measuring the time elapsed
+between receipt of a request
+and its execution.
+
+Took time does *not* include:
+
+* Serializing a request into JSON on a client
+* Sending a request over a network
+* Deserializing a request from JSON on a server
+* Serializing a response into JSON on a server
+* Sending a response over a network
+* Deserializing a response from JSON on a client
+--
+
+
+`timed_out`::
++
+--
+(Boolean)
+If `true`,
+the request timed out before completion;
+returned results may be partial or empty.
+--
+
+`_shards`::
++
+--
+(Object)
+Object containing a count of shards used for the request.
+Returned parameters include:
+
+`total`::
+(Integer)
+Total number of shards involved in the request.
+
+`successful`::
+(Integer)
+Number of shards that executed the request.
+
+`skipped`::
+(Integer)
+Number of shards that skipped the request.
+
+`failed`::
+(Integer)
+Number of shards that failed to execute the request.
+
+--
+
+`hits`::
++
+--
+(Object)
+Contains returned documents and metadata.
+Returned parameters include:
+
+`total`::
+(Object)
+Metadata about the number of returned documents.
+Returned parameters include:
++
+* `value`: Total number of returned documents.
+* `relation`: Indicates whether the number of documents returned.
+Returned values are:
+** `eq`: Accurate
+** `gte`: Lower bound
+
+`max_score`::
+(Float)
+Highest returned document `_score`.
++
+The `_score` parameter is a floating point number
+used to determine the relevance of the returned document.
```

Comment:
Maybe mention that this will be null on requests that don't sort by score?

Reply:
Fixed with 2ed8cdafdf035d6535ce72e9df66fd8f420d3703.